2021-08-14 20:22:55,355 INFO    MainThread:23090 [wandb_setup.py:_flush():69] setting env: {}
2021-08-14 20:22:55,355 INFO    MainThread:23090 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-14 20:22:55,355 INFO    MainThread:23090 [wandb_init.py:_log_setup():337] Logging user logs to /cs/labs/josko/nirm/embryo_project_version1/embyo_projects_codes/lord-pytorch-unet/wandb/run-20210814_202255-11f01m5y/logs/debug.log
2021-08-14 20:22:55,355 INFO    MainThread:23090 [wandb_init.py:_log_setup():338] Logging internal logs to /cs/labs/josko/nirm/embryo_project_version1/embyo_projects_codes/lord-pytorch-unet/wandb/run-20210814_202255-11f01m5y/logs/debug-internal.log
2021-08-14 20:22:55,355 INFO    MainThread:23090 [wandb_init.py:init():370] calling init triggers
2021-08-14 20:22:55,355 INFO    MainThread:23090 [wandb_init.py:init():377] wandb.init called with sweep_config: {}
config: {}
2021-08-14 20:22:55,355 INFO    MainThread:23090 [wandb_init.py:init():419] starting backend
2021-08-14 20:22:55,355 INFO    MainThread:23090 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-08-14 20:22:55,363 INFO    MainThread:23090 [backend.py:ensure_launched():135] starting backend process...
2021-08-14 20:22:55,369 INFO    MainThread:23090 [backend.py:ensure_launched():140] started backend process with pid: 23149
2021-08-14 20:22:55,370 INFO    MainThread:23090 [wandb_init.py:init():424] backend started and connected
2021-08-14 20:22:55,371 INFO    MainThread:23090 [wandb_init.py:init():472] updated telemetry
2021-08-14 20:22:55,371 INFO    MainThread:23090 [wandb_init.py:init():491] communicating current version
2021-08-14 20:22:56,060 INFO    MainThread:23090 [wandb_init.py:init():496] got version response upgrade_message: "wandb version 0.12.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-08-14 20:22:56,060 INFO    MainThread:23090 [wandb_init.py:init():504] communicating run to backend with 30 second timeout
2021-08-14 20:22:56,420 INFO    MainThread:23090 [wandb_init.py:init():529] starting run threads in backend
2021-08-14 20:22:56,486 INFO    MainThread:23090 [wandb_run.py:_console_start():1623] atexit reg
2021-08-14 20:22:56,488 INFO    MainThread:23090 [wandb_run.py:_redirect():1497] redirect: SettingsConsole.REDIRECT
2021-08-14 20:22:56,488 INFO    MainThread:23090 [wandb_run.py:_redirect():1502] Redirecting console.
2021-08-14 20:22:56,489 INFO    MainThread:23090 [wandb_run.py:_redirect():1558] Redirects installed.
2021-08-14 20:22:56,489 INFO    MainThread:23090 [wandb_init.py:init():554] run started, returning control to user process
2021-08-14 20:22:56,516 INFO    MainThread:23090 [wandb_run.py:_atexit_cleanup():1593] got exitcode: 1
2021-08-14 20:22:56,516 INFO    MainThread:23090 [wandb_run.py:_restore():1565] restore
2021-08-14 20:22:58,995 INFO    MainThread:23090 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1139
  total_bytes: 1139
}

2021-08-14 20:22:59,403 INFO    MainThread:23090 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1139
  total_bytes: 1139
}

2021-08-14 20:22:59,503 INFO    MainThread:23090 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1139
  total_bytes: 3532
}

2021-08-14 20:22:59,604 INFO    MainThread:23090 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1139
  total_bytes: 3532
}

2021-08-14 20:22:59,705 INFO    MainThread:23090 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1139
  total_bytes: 3532
}

2021-08-14 20:22:59,806 INFO    MainThread:23090 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1139
  total_bytes: 3532
}

2021-08-14 20:22:59,907 INFO    MainThread:23090 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3532
  total_bytes: 3532
}

2021-08-14 20:23:00,008 INFO    MainThread:23090 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3532
  total_bytes: 3532
}

2021-08-14 20:23:00,109 INFO    MainThread:23090 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3532
  total_bytes: 3532
}

2021-08-14 20:23:00,210 INFO    MainThread:23090 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3532
  total_bytes: 3532
}

2021-08-14 20:23:00,311 INFO    MainThread:23090 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3532
  total_bytes: 3532
}

2021-08-14 20:23:00,412 INFO    MainThread:23090 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3532
  total_bytes: 3532
}

2021-08-14 20:23:00,513 INFO    MainThread:23090 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3532
  total_bytes: 3532
}

2021-08-14 20:23:00,615 INFO    MainThread:23090 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3532
  total_bytes: 3532
}

2021-08-14 20:23:01,005 INFO    MainThread:23090 [wandb_run.py:_wait_for_finish():1715] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3532
  total_bytes: 3532
}

2021-08-14 20:23:01,106 INFO    MainThread:23090 [wandb_run.py:_wait_for_finish():1715] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3532
  total_bytes: 3532
}

2021-08-14 20:23:02,146 INFO    MainThread:23090 [wandb_run.py:_show_files():1937] logging synced files
